tests/testthat/test-metadata.R

```diff
@@ -618,7 +618,9 @@ test_that("xportr_*: Domain is kept in between calls", {
   # Divert all messages to tempfile, instead of printing them
   #  note: be aware as this should only be used in tests that don't track
   #        messages
-  withr::local_message_sink(tempfile())
+  if (requireNamespace("withr", quietly = TRUE)) {
+    withr::local_message_sink(withr::local_tempfile())
```

Comment:
Double colon is mandatory to use for Suggested packages, otherwise `R CMD check` will fail

Reply:
Ah yes - could you make a quick note for this in our wiki of when we use `::`

Reply:
Done!